ruby/google-protobuf.gemspec

```diff
@@ -17,12 +17,12 @@ Gem::Specification.new do |s|
   else
     s.files     += Dir.glob('ext/**/*')
     s.extensions= ["ext/google/protobuf_c/extconf.rb"]
-    s.add_development_dependency "rake-compiler-dock", "= 1.1.0"
+    s.add_development_dependency "rake-compiler-dock", "= 1.2.0"
   end
   s.test_files  = ["tests/basic.rb",
                   "tests/stress.rb",
                   "tests/generated_code_test.rb"]
   s.required_ruby_version = '>= 2.3'
-  s.add_development_dependency "rake-compiler", "~> 1.1.0"
+  s.add_development_dependency "rake-compiler", "~> 1.2.0"
```

Comment:
rake-compiler does not have a 1.2.0 release yet: https://rubygems.org/gems/rake-compiler.  Did you mean to move up to the latest release, 1.1.9?

Reply:
I think I may have just mistakenly changed this. Fixed it back to what it was, as it also should support the latest (not that it's needed I think)